### DISCUSSION TITLE
Improved In-App search

### DISCUSSION THREAD

**Discussion by daskinw** [GitbookIO/community#55] [Feature Requests]
### What's your feature/idea?

Performing in-app search should take you to a dedicated search page with results. The current quick find modal does not display all potential options for the search, does not allow you to filter your search in meaningful ways (e.g. I should be able to search only for content that was updated in the last month or that has been updated by a certain user) and when you type a query in the quick find and hit enter you're taken to the first result rather than a dedicated results page (such as you would in any search engine or lots of other apps).

The Quick Find results often miss content. When we need to make updates we are unable to locate all content containing outdated text because Quick Find does not ever show all results.

### What's your desired outcome?

- In-app search will show all potential options
- Dedicated search results page (vs hitting enter taking you to first result)
- Support facets and filters

### What's the impact of this for you?

- [ ] Critical
- [X] High
- [ ] Medium
- [ ] Low

**Comment by ghost** [comment]
This is a fine suggestion, and will stop Jef Raskin's shade from haunting your repositories (he famously didn't like modals back *before* there was a web).

Seriously, having the option for a search results page, combined with the ability to search across spaces ([see #18](https://github.com/orgs/GitbookIO/discussions/18)) would greatly improve the experience for searching published documents.
